### PR TITLE
Base64Encoder v2

### PR DESCRIPTION
**Pull request by darkushin**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/27235

## Description
Create a new version for Base64Encoder. This version will support the encoding of 2 bytes languages.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CommonScripts/Scripts/Base64EncodeV2/Base64EncodeV2_test.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import pytest
+
+
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/Base64EncodeV2/Base64EncodeV2_test.py:3]: let's add test cases with foreign language as described in the issue. also geramn/spanish will be good

File: Packs/CommonScripts/Scripts/Base64EncodeV2/Base64EncodeV2.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  id: Base64EncodeV2
+  version: -1
+enabled: false
+name: Base64EncodeV2
+outputs:
+- contextPath: Base64.encoded
+  description: input encoded as Base64 format.
+  type: Unknown
+script: '-'
+subtype: python3
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/Base64EncodeV2/Base64EncodeV2.yml:19]: no dockerimage?

File: Packs/CommonScripts/Scripts/Base64EncodeV2/Base64EncodeV2.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+comment: Encodes an input to Base64 format.
+commonfields:
+  id: Base64EncodeV2
+  version: -1
+enabled: false
+name: Base64EncodeV2
+outputs:
+- contextPath: Base64.encoded
+  description: input encoded as Base64 format.
+  type: Unknown
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/Base64EncodeV2/Base64EncodeV2.yml:17]: String?

File: Packs/CommonScripts/Scripts/Base64EncodeV2/Base64EncodeV2.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+            'encoded': res
+        }
+    }
+
+    return res, outputs
+
+
+if __name__ in ('builtins', '__builtin__'):
+    input = demisto.args().get('input')
+    return_outputs(*encode(input))
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/Base64EncodeV2/Base64EncodeV2.py:22]: shouldn't you add try/catch for later errors not showing the stacktrace ina warrroom entry?

**Comment by kirbles19** [comment]
@darkushin Done.

**Pull request merged**